packages/react-conformance/src/isConformant.ts

```diff
@@ -63,3 +69,40 @@ export function isConformant<TProps = {}>(...testInfo: Partial<IsConformantOptio
     }
   });
 }
+
+/**
+ * Run default and extra tests that don't require TypeScript information
+ * @param mergedOptions
+ */
+function runNonTypeTests(mergedOptions: IsConformantOptions) {
```

Comment:
This is a bit more complicated than it needs to be IMO:

FYI double check the filter conditions.

```ts
const allTests = Object.entries(defaultTests).concat(Object.entries(extraTests ?? {}));

      const testsToRun = Object.entries(allTests).filter(([testName, testFn]) => {
        return !disabledTests.includes(testName) ||(disbleTypescriptTests && testFn.length === 1);
      });

      for(const testToRun of testsToRun) {
        testToRun();
      }
```

Regarding the componentInfo and tsProgram - you can refactor the tests to all use a lazy getter to initialize them